src/client/routes/index.js

```diff
@@ -58,6 +59,7 @@ export default (store) => {
     store.dispatch(fetchPullRequestMetadata(id))
     store.dispatch(fetchPullRequestDiscussion(id))
     store.dispatch(fetchPullRequestFilesList(id))
+    store.dispatch(fetchPullRequestFiles(id))
```

Comment:
This may as much be a GraphQL API clarity thing, but giving an empty string for the `name` variable fetches all files?

Also, for later, should we not try to fetch them file-by-file on demand instead? Some of the PRs are quite heavy. This could perhaps be combined with caching the diffs serverside (Ono)?